components/playlist/browser/resources/components/playlistInfo.tsx

```diff
@@ -23,44 +26,44 @@ export const PlaylistInfoContainer = styled.div`
   column-gap: 8px;
 `
 
-export const PlaylistName = styled.div`
+export const PlaylistName = styled.div<{ color?: string }>`
   height: 28px;
   grid-row: 1 / 2;
   grid-column: 1 / 4;
   align-self: flex-end;
+  ${p => p.color && css`p.color`}
```

Comment:
Does this work?  I thought you needed to specify the property too (if it works, that's awesome!)

```suggestion
  ${p => p.color && css`color: p.color`}
```